From 8b6312204485972e5de4c39d2397cd182ce82953 Mon Sep 17 00:00:00 2001
From: Marko Katic <dromede.gmail.com>
Date: Fri, 16 Mar 2012 00:31:02 +0100
Subject: [PATCH 5/5] xcb-util-wm: Add recipe that provides libxcb-icccm and libxcb-ewmh

Xcb-util was split into separate packages in the 0.3.6 -> 0.3.8 transition.
These libraries were previously provided by xcb-util and are now a separate package.
---
 meta/recipes-graphics/xcb/xcb-util-wm_0.3.8.bb |   12 ++++++++++++
 1 files changed, 12 insertions(+), 0 deletions(-)
 create mode 100644 meta/recipes-graphics/xcb/xcb-util-wm_0.3.8.bb

diff --git a/meta/recipes-graphics/xcb/xcb-util-wm_0.3.8.bb b/meta/recipes-graphics/xcb/xcb-util-wm_0.3.8.bb
new file mode 100644
index 0000000..91f9a81
--- /dev/null
+++ b/meta/recipes-graphics/xcb/xcb-util-wm_0.3.8.bb
@@ -0,0 +1,12 @@
+require xcb-util.inc
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://ewmh/ewmh.c;endline=27;md5=63fd80552d7dc886f4eb267610d5bb9d \
+					file://ewmh/xcb_ewmh.h;startline=4;endline=30;md5=7bcd2d64f5615707f95ea8826a93561f \
+					file://icccm/icccm.c;endline=28;md5=e2b3240a0c197e8977adde2a5cf18d50 \
+					file://icccm/xcb_icccm.h;startline=4;endline=31;md5=0af3f37474085b0517dbbae4cb4ce13c \
+					"
+PR = "r0"
+
+SRC_URI[md5sum] = "dda47289bc846a6a3e07824e9ec3aef8"
+SRC_URI[sha256sum] = "c557abbe4c7d45268f0541ee3d8f9a1a8799692fcc28aa33f39db93cc2e65007"
-- 
1.7.4.1

